take_a_break.py

```diff
@@ -9,6 +9,7 @@
 t=hr*3600+minute*60+sec
 total=int(raw_input("total no. of breaks?->"))
 flag=0
+//i="commitchanges"
```

Comment:
instead of // use # to make it in form of a comment

Reply:
ookk